spec/models/folio/patron_spec.rb

```diff
@@ -50,30 +50,65 @@
     it { is_expected.to be_fee_borrower }
   end
 
-  describe '#good_standing?' do
+  describe '#make_request_as_patron?' do
     let(:folio_client) { instance_double(FolioClient, patron_blocks:) }
-    let(:patron_blocks) { { 'automatedPatronBlocks' => [] } }
+    let(:patron_blocks) do
+      { 'automatedPatronBlocks' => [
+        {
+          patronBlockConditionId: 'ac13a725-b25f-48fa-84a6-4af021d13afe',
+          blockBorrowing: false,
+          blockRenewals: false,
+          blockRequests: true,
+          message: 'Patron has reached maximum allowed outstanding fee/fine balance for his/her patron group'
+        }
+      ] }
+    end
 
     before do
       allow(described_class).to receive(:folio_client).and_return(folio_client)
     end
 
-    it { is_expected.to be_good_standing }
+    it { is_expected.to be_make_request_as_patron }
 
-    context 'when blocks exist' do
-      let(:patron_blocks) do
-        { 'automatedPatronBlocks' => [
-          {
-            patronBlockConditionId: 'ac13a725-b25f-48fa-84a6-4af021d13afe',
-            blockBorrowing: false,
-            blockRenewals: false,
-            blockRequests: true,
-            message: 'Patron has reached maximum allowed outstanding fee/fine balance for his/her patron group'
-          }
-        ] }
+    context 'not active' do
+      let(:fields) do
+        {
+          'username' => 'jcoyne85',
+          'id' => '562a5cb0-e998-4ea2-80aa-34ac2b536238',
+          'externalSystemId' => '09658748',
+          'barcode' => '2555716958',
+          'active' => false,
+          'patronGroup' => '985acbb9-f7a7-4f44-9b34-458c02a78fbc',
+          'departments' => [],
+          'proxyFor' => [],
+          'personal' =>
+            { 'lastName' => 'Coyne',
+              'firstName' => 'Justin',
+              'middleName' => 'Michael',
+              'email' => 'foliotesting@lists.stanford.edu',
+              'phone' => '(612) 868-2411',
+              'addresses' =>
+              [{ 'countryId' => 'US',
+                 'addressLine1' => '4621 Ewing Ave S',
+                 'city' => 'Minneapolis',
+                 'region' => 'Minnesota',
+                 'postalCode' => '55410-1745',
+                 'addressTypeId' => '93d3d88d-499b-45d0-9bc7-ac73c3a19880',
+                 'primaryAddress' => true }],
+              'preferredContactTypeId' => '002' },
+          'enrollmentDate' => '2022-09-01T00:00:00.000+00:00',
+          'createdDate' => '2023-05-20T00:13:20.324+00:00',
+          'updatedDate' => '2023-05-20T00:13:20.324+00:00',
+          'metadata' =>
+            { 'createdDate' => '2023-02-10T02:14:56.209+00:00',
+              'createdByUserId' => '3e2ed889-52f2-45ce-8a30-8767266f07d2',
+              'updatedDate' => '2023-05-20T00:13:20.249+00:00',
+              'updatedByUserId' => '9c77fffb-9d40-422c-8742-4c0f79957888' },
+          'customFields' => { 'affiliation' => 'staff', 'proximitychipid' => '0577826' }
```

Comment:
not sure what `proximitychipid` is exactly, but are you sure you want it public?

Reply:
same for address, barcode, etc.

Reply:
Thanks, redacted.